yarn.lock

```diff
@@ -5920,7 +5930,7 @@ spectacle@5.2.2:
     normalize.css "^7.0.0"
     prismjs "1.6.0"
     react-emotion "^8.0.8"
-    react-live "^1.11.0"
+    react-live "^1.12.0"
```

Comment:
Hey Wendy! This all looks great, but I am going to make a small change and revert to the previous version of `react-live` - there's a weird issue where the version always has to be the same as whatever `prismjs` is using, so I want to keep that aligned. Once I've made that change I'll merge it - just wanted to give you the context there!

Reply:
Oh ok, thanks!